StyleCop.Analyzers/StyleCop.Analyzers/OrderingRules/SA1206DeclarationKeywordsMustFollowOrder.cs

```diff
@@ -96,6 +96,12 @@ private static void CheckModifiersOrderAndReportDiagnostics(SyntaxNodeAnalysisCo
             foreach (var modifier in modifiers)
             {
                 var currentModifierType = GetModifierType(modifier);
+                if (currentModifierType == ModifierType.None)
+                {
+                    // Just ignore unknown modifiers, to avoid false positives before support for
+                    // new language features has been added.
+                    continue;
+                }
```

Comment:
💭 I almost prefer the original behavior, otherwise support may never have been added.

Reply:
I think that "never" is unlikely, but it would almost certainly have taken longer to be discovered. I was just trying to think about the user experience and for my part I would definitely prefer false negatives over false positives, but I can revert that change if you want me to. Just say the word.

Reply:
@sharwell Could you please make you call here? Thank you.

Reply:
Ok. I reverted this change now.

Reply:
Imo behavior proposed by @bjornhellander is much better. We have required keyword for >6 months. This means users of stylecop either do not use SA1206 or do not use required keyword over this period.

Reply:
@POnakS We do use it (_required keyword_) but we have to constantly disable SA1206 by pragma.